planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -560,6 +586,38 @@ void ObstacleAvoidancePlanner::publishDebugData(const Header & header) const
 
   time_keeper_ptr_->toc(__func__, "  ");
 }
+
+void ObstacleAvoidancePlanner::transformPathToOdomFrame(Path::SharedPtr path_ptr) const {
+    geometry_msgs::msg::TransformStamped transform_stamped =
+            map_odom_tf_buffer_->lookupTransform("odom", "map", tf2::TimePointZero);
```

Comment:
If this transform is unavailable, this will throw an error and kill the node. Please wrap with try/catch.

Reply:
I have seen two ways to do this: a. use try/catch; b. do the canTransform prior to lookupTransform. I used option b and I am okay with both. Do you think I should change it to try/catch.

Reply:
I get it now. Thanks.